microsoft/testsuites/core/vm_hot_resize.py

```diff
@@ -37,14 +39,30 @@ class VmHotResize(TestSuite):
             supported_features=[Resize],
         ),
     )
-    def verify_vm_hot_resize(self, node: Node) -> None:
+    def verify_vm_hot_resize(self, log: Logger, node: Node) -> None:
         resize = node.features[Resize]
-        expected_vm_capability = resize.resize()
+        retry = 1
+        maxretry = 10
+        while retry < maxretry:
+            try:
+                expected_vm_capability = None
+                expected_vm_capability = resize.resize()
+                break
+            except Exception as e:
```

Comment:
In other places, we use `identifier` as the name.

Reply:
update